feature/bgp/policybase/otg_tests/extcomm_action_test/README.md

```diff
@@ -0,0 +1,176 @@
+# RT-7.8: BGP Policy - Set Communities
+
+## Summary
+
+Configure bgp policy to add communities to routes by matching on the following
+criteria.
+
+* RT-7.8.2 Validate policy to set standard community for all routes using OC release 2.x
+* RT-7.8.3 Validate policy to set ext-community for various match criteria using OC release 2.x
+* RT-7.8.4 Validate community-sets and routing-policy using OC release 3.x
+* RT-7.8.5 Validate ext-community-sets and routing-policy using OC release 3.x
+
+## Testbed type
+
+* https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+* Testbed configuration - Setup BGP sessions and prefixes.
+  * Generate config for 2 DUT and ATE ports where:
+    * DUT port 1 to ATE port 1.
+    * DUT port 2 to ATE port 2.
+  * Configure ATE port 1 with a BGP session to DUT port 1.
+    * Advertise ipv4 and ipv6 prefixes to DUT port 1 using the following communities:
+      * prefix-set-1 with 2 routes without communities.
+      * prefix-set-2 with 2 routes with communities `[5:5, 6:6 ]`.
+      * prefix-set-3 with 2 routes with extended communities `[50:500000, 60:600000]`.
```

Comment:
"extended communities `[50:500000, 60:600000]`."

This are illegal values!
- They are not standard communities, because Local admin part is >65535
- They are not large comunities (rfc8092) because violates chapter 5 of this rfc
- They are not extended communities, because extended community MUST have designated type

this is blocking